context/app/static/js/shared-styles/panels/IconPanel/IconPanel.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { PropsWithChildren } from 'react';
+
+import Stack from '@mui/material/Stack';
+import Paper from '@mui/material/Paper';
+import Typography from '@mui/material/Typography';
+
+import SeverityIcon, { ColoredStatusIconProps } from 'js/shared-styles/icons/SeverityIcon';
+
+interface IconPanelProps extends PropsWithChildren {
+  $iconStatus: ColoredStatusIconProps['$iconStatus'];
```

Comment:
We start props with `$` if they are only used in styling. Can we reuse the `IconStatus` type in `SeverityIcon` here? 

```
{status: IconStatus}
```